src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -6076,469 +6049,354 @@ void DynamicIntConvSurfaceClassification(EnergyPlusData &state, int const SurfNu
         }
     }
 
+    constexpr std::array<std::array<ConvectionConstants::InConvClass, ConvectionConstants::NumConvSurfDeltaT>,
+                         ConvectionConstants::NumSurfConvOrientation>
+        A1{{
```

Comment:
LOVE IT! Would it be clearer to initialize these using array of array notation, e.g., `{{ , }, { , }, {, }};` rather than as a flat array?

Reply:
I'll give it a shot. I looked at a few different ways of initializing these but never quite found one that seemed intuitive.